lms/urls.py

```diff
@@ -267,6 +267,11 @@
             'courseware.module_render.xblock_resource',
             name='xblock_resource_url'),
 
+        # xblock Rendering
+        url(r'^xblock/{usage_key_string}$'.format(usage_key_string=settings.USAGE_KEY_PATTERN),
```

Comment:
@andy-armstrong This change in the URL is a direct response to the conversation we had in the Arch council about making the xBlock rendering a URL rather than an API.  I believe you were strongly (understandably) against returning a rendered HTML student view page as an API response.  The perspective was to treat this as an HTML view rather than an API transaction.